src/libp2p/Libp2p.Protocols.Noise.Tests/NoiseProtocolTests.cs

```diff
@@ -31,7 +32,41 @@ public async Task Test_ConnectionEstablished_AfterHandshake()
         // channelFactory.Received().SubDialAndBind(downChannelFromProtocolPov, peerContext, proto1);
         // await downChannel.CloseAsync();
     }
-
+    [Test]
+    public async Task Test_ConnectionEstablished_With_PreSelectedMuxer()
+    {
```

Comment:
Could you add an empty line between [AAA parts](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) of the test